Automattic/wp-calypso
# Decorate domains in domain-only sites with parked status in the domain settings

#### Changes proposed in this Pull Request

* Domains in domain-only sites are now referred to as `Parked Domain` in  the domain settings view in lieu of `Registered Domain`.
* A context-sensitive notice is displayed with special status.
* CTA buttons are added to ease working with this domain.

#### Testing instructions



* Ensure that the domain management settings view for a domain in a domain-only site looks like the following:

<img width="742" alt="Screenshot 2020-07-17 at 10 51 37 PM" src="https://user-images.githubusercontent.com/277661/87833592-2035fe80-c880-11ea-9afd-6882f592053a.png">

* Ensure that the `Change name servers` button works
